lib/ui/views/onboarding_view.dart

```diff
@@ -24,58 +26,75 @@ class _OnBoardingViewState extends State<OnBoardingView> {
         MediaQuery.of(context).size.height - marginFromSafeArea;
     return Scaffold(
       body: SafeArea(
-          child: Container(
-        child: Column(
-          children: <Widget>[
-            Container(
-              height: heightOfScreen * 0.6,
-              child: PageView(
-                children: <Widget>[
-                  _buildWalkThrough(),
-                  _buildWalkThrough(),
-                  _buildWalkThrough()
-                ],
-                onPageChanged: (value) {
-                  setState(() => currentIndexPage = value.toDouble());
-                },
-              ),
-            ),
-            Container(
-              height: heightOfScreen * 0.1,
-              child: DotsIndicator(
-                dotsCount: pageLength,
-                position: currentIndexPage,
-                decorator: DotsDecorator(
-                  color: Colors.blue,
-                  activeColor: Colors.red,
+        child: Container(
+          child: Column(
+            children: <Widget>[
+              Flexible(
+                child: Container(
+                  child: PageView(
+                    children: <Widget>[
+                      _buildWalkThrough(context, 'undraw_welcome_3gvl.svg',
+                          "Welcome to DigiPol! Where digital direct democracy starts."),
+                      _buildWalkThrough(context, 'undraw_process_e90d.svg',
+                          "DigiPol collates all the current bills in the Australian goverment for you."),
+                      _buildWalkThrough(context, 'undraw_voting_nvu7.svg',
+                          "So you can stay informed and vote directly on what matters most."),
+                      _buildWalkThrough(context, 'undraw_new_ideas_jdea.svg',
+                          "You can even vote on current issues created by the community or submit your own."),
+                      _buildWalkThrough(context, 'undraw_ethereum_fb7n.svg',
+                          "We don't store any of your information, and all votes are verified by the ethereum blockchain."),
```

Comment:
Um. Who's "we"

Reply:
All copy was just place holder really. This could  just be changed to 'digipol'

Reply:
I shouldn't have said that, and instead said something clearer: you're making a strong promise on behalf of more than just yourself, and ambiguously at that. Isn't a vote 'their information' as such?

Reply:
I was trying to just explain simply what happens with any data (voting, names etc). I agree, too ambiguous. What is the best wording for this?

Reply:
If you remove copy as part of this PR, it doesn't matter.